python/src/main/java/org/navgurukul/playground/editor/PythonEditorActivity.kt

```diff
@@ -34,6 +37,17 @@ class PythonEditorActivity : AppCompatActivity() {
     override fun onCreateOptionsMenu(menu: Menu?): Boolean {
         val inflater: MenuInflater = menuInflater
         inflater.inflate(R.menu.python_playground_menu, menu)
+        for (i in 0 until menu!!.size()) {
```

Comment:
instead of ' menu!! ' you should use menu?.let{ }
This ensures that the code won't run if the menu is null. However, in the earlier case it could crash.
Also doing after this you wouldn't require to use ' menu!! ' in the below code as 'it' would be available in the scope